test/core/simd/simd_i8x16.wast

```diff
@@ -0,0 +1,387 @@
+                                     (v128.const i8x16 +0 +0 +0 +0 0 0 0 0 -0 -0 -0 -0 0 0 0 0))
+                                     (v128.const i8x16 -0 -0 -0 -0 -0 -0 -0 -0 +0 +0 +0 +0 +0 +0 +0 +0))
+(assert_return (invoke "i8x16.max_s" (v128.const i8x16 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0)
+                                     (v128.const i8x16 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0))
+                                     (v128.const i8x16 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0))
+(assert_return (invoke "i8x16.max_u" (v128.const i8x16 -0 -0 -0 -0 -0 -0 -0 -0 +0 +0 +0 +0 +0 +0 +0 +0)
+                                     (v128.const i8x16 +0 +0 +0 +0 0 0 0 0 -0 -0 -0 -0 0 0 0 0))
+                                     (v128.const i8x16 -0 -0 -0 -0 -0 -0 -0 -0 +0 +0 +0 +0 +0 +0 +0 +0))
+(assert_return (invoke "i8x16.max_u" (v128.const i8x16 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0)
+                                     (v128.const i8x16 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0 +0))
+                                     (v128.const i8x16 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0 -0))
+
+;; Type check
+(assert_invalid (module (func (result v128) (i8x16.min_s (i32.const 0) (f32.const 0.0)))) "type mismatch")
+(assert_invalid (module (func (result v128) (i8x16.min_u (i32.const 0) (f32.const 0.0)))) "type mismatch")
+(assert_invalid (module (func (result v128) (i8x16.max_s (i32.const 0) (f32.const 0.0)))) "type mismatch")
+(assert_invalid (module (func (result v128) (i8x16.max_u (i32.const 0) (f32.const 0.0)))) "type mismatch")
+
+;; Test operation with empty argument
+(assert_invalid
+  (module
+    (func $i8x16.min_s-1st-arg-empty (result v128)
+      (i8x16.min_s (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0))
+    )
+  )
+  "type mismatch"
+)(assert_invalid
+  (module
+    (func $i8x16.min_s-all-args-empty (result v128)
+      (i8x16.min_s)
+    )
+  )
+  "type mismatch"
+)(assert_invalid
+  (module
+    (func $i8x16.min_u-1st-arg-empty (result v128)
+      (i8x16.min_u (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0))
+    )
+  )
+  "type mismatch"
+)(assert_invalid
+  (module
+    (func $i8x16.min_u-all-args-empty (result v128)
+      (i8x16.min_u)
+    )
+  )
+  "type mismatch"
+)(assert_invalid
+  (module
+    (func $i8x16.max_s-1st-arg-empty (result v128)
+      (i8x16.max_s (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0))
+    )
+  )
+  "type mismatch"
+)(assert_invalid
+  (module
+    (func $i8x16.max_s-all-args-empty (result v128)
+      (i8x16.max_s)
+    )
+  )
+  "type mismatch"
+)(assert_invalid
+  (module
+    (func $i8x16.max_u-1st-arg-empty (result v128)
+      (i8x16.max_u (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0))
+    )
+  )
+  "type mismatch"
+)(assert_invalid
+  (module
+    (func $i8x16.max_u-all-args-empty (result v128)
+      (i8x16.max_u)
+    )
+  )
+  "type mismatch"
+)
+
+;; Combination
```

Comment:
How are we deciding what combinations of instructions are useful to test?

Reply:
Good question, actually I don't have a specific definition, I just choose some combinations randomly.

And we don't want to leverage full set of pairwise comparison, since it will cause exponential test cases with low efficiency to capture bugs.

Reply:
Sounds good. I would err on the side of testing fewer combinations because I agree that we want to avoid having more tests than necessary and I don't think the combinations will find as many bugs.